advanced/Scripts/update.sh

```diff
@@ -19,74 +19,17 @@ readonly ADMIN_INTERFACE_DIR="/var/www/html/admin"
 readonly PI_HOLE_GIT_URL="https://github.com/pi-hole/pi-hole.git"
 readonly PI_HOLE_FILES_DIR="/etc/.pihole"
 
-is_repo() {
-  # Use git to check if directory is currently under VCS, return the value
-  local directory="${1}"
-  local curdir
-  local rc
-
-  curdir="${PWD}"
-  cd "${directory}" &> /dev/null || return 1
-  git status --short &> /dev/null
-  rc=$?
-  cd "${curdir}" &> /dev/null || return 1
-  return "${rc}"
-}
+PH_TEST=true
```

Comment:
Sets the environment variable to true so that the basic-install.sh doesn't autostart the `main` function. Lets the basic-install.sh be sourced quietly.